### ISSUE TITLE
Update default data sources

### ISSUE THREAD

**Issue by raymondben** [AustralianAntarcticDivision/raadtools#146]
Consider updating the default data set choices. e.g. `readtopo()` defaults to "gebco_08", which is dated now to say the least. Users might well be unaware that this is what they are being given. It would probably be better to keep the default here to whatever the most recent GEBCO product is.
Other reader functions might be in a similar situation.

**Comment by mdsumner** [comment]
yep good one, readtopo fixed: 14e2f01ab662b02a0eb6bacf2b6085516ef276b3

**Comment by Maschette** [comment]
similar but different, the kerguelen dataset listed: 
```
Kerguelen Plateau Bathymetric Grid, https://data.gov.au/dataset/ds-ga-a05f7893-007f-7506-e044-00144fdd4fa6/details?q=
```
has been superseded by this one: 
https://ecat.ga.gov.au/geonetwork/srv/eng/catalog.search#/metadata/147703 

if possible could we get it updated?

**Comment by mdsumner** [comment]
(kerg bathy added to blueant, where we list our specific datasets of interest for bowerbird: https://github.com/AustralianAntarcticDivision/blueant/commit/b1b054e4a14e8152446e784060262faa56ebce19)
